lib/ansible/modules/network/nxos/nxos_vlan.py

```diff
@@ -318,6 +319,9 @@ def map_obj_to_commands(updates, module):
 
     if purge:
         for h in have:
+            if h['vlan_id'] == '1':
+                # Deletion of vlan 1 is not allowed
+                continue
```

Comment:
I suggest you generate a warning and then continue so that it's clear to users that this is intentional.

Reply:
+1

Reply:
Done. Output from playbook test:

```
ok: [n6k-77.cisco.com] => {
    "msg": {
        "changed": false,
        "commands": [],
        "failed": false,
        "warnings": [
            "Deletion of vlan 1 is not allowed; purge will ignore vlan 1"
        ]
    }
}
```